product_logistics_uom/models/product_template.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Akretion (https://www.akretion.com).
+# @author Raphaël Reverdy <raphael.reverdy@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    volume_uom_id = fields.Many2one(
+        "uom.uom",
+        string="Volume Unit of Measure",
+        domain=lambda self: [
+            ("category_id", "=", self.env.ref("uom.product_uom_categ_vol").id)
+        ],
+        default=lambda x: x._get_volume_uom_id_from_ir_config_parameter(),
+    )
+    volume_uom_name = fields.Char(
+        string="Volume unit of measure label",
+        related="volume_uom_id.name",
+        readonly=True,
+    )
+
+    weight_uom_id = fields.Many2one(
+        "uom.uom",
+        string="Weight Unit of Measure",
+        domain=lambda self: [
+            ("category_id", "=", self.env.ref("uom.product_uom_categ_kgm").id)
+        ],
+        compute=False,
+        default=lambda x: x._get_weight_uom_id_from_ir_config_parameter(),
```

Comment:
Same comment -> it is better to use self instead of one character variable

Reply:
Done